app/scripts/metamask-controller.js

```diff
@@ -1090,6 +1091,16 @@ export default class MetamaskController extends EventEmitter {
       this.submitPassword(password);
     }
 
+    // Log them in if possible
+    if (isManifestV3()) {
+      chrome.storage.local.get(['decryptedKey']).then(({ key }) => {
```

Comment:
I'm not convinced this is the best place for this call, but parking it here for now, since we attempt password submission right above it.

Reply:
this should be `chrome.storage.session` right?